src/components/carousel/_index.scss

```diff
@@ -0,0 +1,187 @@
+@use "../../scss";
+@use "~slick-carousel/slick/slick";
```

Comment:
Do we need to bring in this CSS file if we are defining so much below?

Reply:
Yes. it's the base and I want it in sync with the release

Reply:
What is it bringing in that is not defined in this file? And by importing it this way we are unable to overwrite those styles

Reply:
We agreed yesterday that we wouldn't freese react-slick. Instead, we would import the base scss file from slick-carousel but copy in the theme file from that package so that we can integrate it into the advanced styling.